package.json

```diff
@@ -37,7 +37,12 @@
       "TWEEN": "readonly",
       "webkitAudioContext": "readonly",
       "XRRay": "readonly",
-      "XRWebGLLayer": "readonly"
+      "XRWebGLLayer": "readonly",
+      "GPUBufferUsage": "readonly",
+      "GPUTextureUsage": "readonly",
+      "GPUShaderStage": "readonly",
+      "GPUColorWrite": "readonly",
+      "glslang": "readonly"
```

Comment:
The list is alphabetically ordered. Can you slot them in in the right place please?

Reply:
done, I missed that.